cli/endpoints/online/managed/vnet/setup/modules/machinelearning.bicep

```diff
@@ -0,0 +1,74 @@
+// Creates a machine learning workspace, private endpoints and compute resources
+
+@description('Azure region of the deployment')
+param location string
+
+@description('Tags to add to the resources')
+param tags object
+
+@description('Machine learning workspace name')
+param machineLearningName string
+
+@description('Machine learning workspace display name')
+param machineLearningFriendlyName string = machineLearningName
+
+@description('Machine learning workspace description')
+param machineLearningDescription string
+
+@description('Resource ID of the application insights resource')
+param applicationInsightsId string
+
+@description('Resource ID of the container registry resource')
+param containerRegistryId string
+
+@description('Resource ID of the key vault resource')
+param keyVaultId string
+
+@description('Resource ID of the storage account resource')
+param storageAccountId string
+
+@description('Resource ID of the subnet resource')
+param subnetId string
+
+@description('Resource ID of the virtual network')
+param virtualNetworkId string
+
+@description('Machine learning workspace private link endpoint name')
+param machineLearningPleName string
+
+resource machineLearning 'Microsoft.MachineLearningServices/workspaces@2021-04-01' = {
+  name: machineLearningName
+  location: location
+  tags: tags
+  identity: {
+    type: 'SystemAssigned'
+  }
+  properties: {
+    // workspace organization
+    friendlyName: machineLearningFriendlyName
+    description: machineLearningDescription
+
+    // dependent resources
+    applicationInsights: applicationInsightsId
+    containerRegistry: containerRegistryId
+    keyVault: keyVaultId
+    storageAccount: storageAccountId
+
+    allowPublicAccessWhenBehindVnet: false
```

Comment:
Is this the one that set PNA to false?

Reply:
thanks for the catch. clarified with wrp team - changing to publicNetworkAccess flag now